static/js/containers/pages/login/LoginPasswordPage.js

```diff
@@ -50,11 +50,11 @@ class LoginPasswordPage extends React.Component<Props> {
 
     try {
       const {
-        body: { state, redirectUrl, errors }
+        body: { state, redirect_url, errors }
       }: { body: AuthResponse } = await loginPassword(password, partialToken)
 
       if (state === STATE_SUCCESS) {
-        window.location.href = redirectUrl || routes.dashboard
+        window.location.href = redirect_url || routes.dashboard
```

Comment:
@pdpinch I am able to fix the `redirect_url` issue, not sure it thats the correct way of fixing it. I would like @gsidebo to take a look at that particular change.

Reply:
Is this the same issue https://github.com/mitodl/mitxpro/issues/542 ?

If it is, please open a PR to just fix #542.

Reply:
Yes, I will open PR in a while.

Reply:
Removed those changes from this PR and created new one for redirect url fix #658.